app/test_main.py

```diff
@@ -1,3 +1,47 @@
 from app.main import get_human_age
 
-# write your code here
+import pytest
+
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected_result", [
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="under lower boundary condition"
+            ),
+            pytest.param(
+                15,
+                23,
```

Comment:
In this case, you test one value only for the cat but not for the dog, the same in other cases. You should check all age boundaries for both of them.